src/test/ui/rustdoc/renamed_features.rs

```diff
@@ -0,0 +1,5 @@
+#![feature(doc_keyword)] //~ ERROR
```

Comment:
Could you rename this test to something a bit more descriptive, that includes rustdoc_internals in it?